apps/i18n/common/en_us.json

```diff
@@ -13,6 +13,8 @@
   "and": "and",
   "animationMode": "Animation",
   "announcements": "Announcements",
+  "announcementHeadingCsfAtoF": "Courses 1-4 updated to align with grades K-5",
+  "announcementDescriptionCsfAtoF": "We've tailored the courses to each grade level and re-dubbed them Courses A-F.",
   "announcementHeadingCspScores": "AP CS Principles Scores",
   "announcementDescriptionCspScores": "Over 70% of students earned passing scores in the first ever CS Principles exam.",
```

Comment:
Should I delete these? Or do they stick around for some sort of announcements posterity?